### ISSUE TITLE
[DynamoDB] Table.GetItem does not support bool type.

### ISSUE THREAD

**Issue by anphung** [crowdmob/goamz#328]
It omits boolean key-value pair.
Here:
github.com/crowdmob/goamz/dynamodb/item.go: func parseAttributes

**Comment by alimoeeny** [comment]
Yeah, as far as I know, only `S` (string) type is supported. Read this: https://github.com/crowdmob/goamz/issues/308
